pages/osx/xcrun.md

```diff
@@ -0,0 +1,27 @@
+# xcrun
+
+> Run or locate development tools and properties.
+
+- Get help:
+
+`xcrun --help`
+
+- Get version:
+
+`xcrun --version`
+
+- Find and run a tool from the active developer directory:
+
+`xcrun {{tool}} {{args}}`
+
+- Show verbose output:
+
+`xcrun --verbose`
```

Comment:
Does this example do anything on its own?

Reply:
No, it doesn't. Thanks for pointing that out. I've fixed it.